### PR TITLE
Add to contributors, revise to version 0.2

### PR DESCRIPTION
**Pull request by dshkol**
- Added @atheriel as an author
- Revised version count to 0.2 to reflect additions of geo and vector list/search, even if in early form - these are significant changes
- Revise description slightly
- Removed tmap as a suggest. It's kind of a novelty package to do one particular thing (but it does it well).

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -29,8 +29,7 @@ Suggests: knitr,
     rgeos,
     sf,
     sp,
-    tidyr,
-    tmap
+    tidyr
```
Comment (atheriel) [inline DESCRIPTION]: I don't have an issue with removing `tmap`, but we'll have to remove it from the vignettes as well. I guess the plan is to replace it with `ggplot2` code?

File: DESCRIPTION [hunk 2]
```diff
@@ -4,11 +4,11 @@ Title: CensusMapper API Interface to Canada Census
 Version: 0.1.0
 Authors@R: c(
     person("Jens", "von Bergmann", email = "jens@mountainmath.ca", role = c("aut", "cre")),
-    person("Dmitry", "Shkolnik", email = "shkolnikd@gmail.com", role = "aut"))
+    person("Dmitry", "Shkolnik", email = "shkolnikd@gmail.com", role = "aut"),
+    person("Aaron","Jacobs", email = "atheriel@gmail.com", role = "aut"))
 Description: The package provides convenient access to StatCan census data via CensusMapper APIs.
```
Comment (atheriel) [inline DESCRIPTION]: :smile:

File: DESCRIPTION (hunk 1 above)
Comment (dshkol) [inline DESCRIPTION]: That would be my preference. We can do ad hoc examples with other approaches (tmap, leaflet, htmlwidgets, etc.) but they don't need to be packaged vignettes.

Comment (atheriel) [inline DESCRIPTION]: Sounds good to me.

Comment (atheriel) [inline DESCRIPTION]: I'll file a separate issue.

**Pull request merged**